docs/releases.md

```diff
@@ -0,0 +1,58 @@
+* [Introduction](#introduction)
+* [Versioning](#versioning)
+* [Tagging repositories](#tagging-repositories)
+* [Release process](#release-process)
+* [Releases](#releases)
+
+## Introduction
+
+This document provides details about Gluster CSI driver releases.
+
+## Versioning
+
+The Gluster CSI driver project uses [semantic versioning](http://semver.org/) for all releases. Semantic versions are comprised of three fields in the form:
+
+```
+MAJOR.MINOR.PATCH
+```
+
+For examples: `1.0.0`, `1.0.0-rc.2`.
+
+Semantic versioning is used since the version number is able to convey clear information about how a new version relates to the previous version. For example, semantic versioning can also provide assurances to allow users to know when they must upgrade compared with when they might want to upgrade:
+
+- When `PATCH` increases, the new release contains important **security fixes**
+  and an upgrade is recommended.
+
+  The patch field can contain extra details after the number. Dashes denote pre-release versions. `1.0.0-rc.2` in the example denotes the fifth release candidate for release `1.0.0`.
+  
+- When `MINOR` increases, the new release adds **new features** but *without
+  changing the existing behavior*.
+
+- When `MAJOR` increases, the new release adds **new features, bug fixes, or
+  both** and which *changes the behavior from the previous release* (incompatible with previous releases).
+
+  A major release will also likely require a change of the container manager version used, for example Docker\*. Please refer to the release notes for further details.
```

Comment:
I don't understand this item.

Reply:
Basically I wanted to say the tag of docker image has to be available with the version number. As its a general process, I am skipping this line.